learn/glossary.md

```diff
@@ -73,16 +129,62 @@ A static URI (or IP) that you have assigned to your node, which applications can
+
+## Trophy
+
+Achievement earned for proving knowledge of and participation in the Pocket Network ecosystem. These can be collected and used to earn a vote in the Pocket DAO. There are [four distinct paths to a vote](https://docs.pokt.network/home/community/trophies), each with different trophies: App Developers, Node Runners, Community Shepherds, and Governors/Contributors.
+
+## v0
+
+Current active iteration of the Pocket protocol. Often used in contrast to the upcoming v1 update.
+
+## v1
+
+Pocket v1 (or just "v1") is a [major planned overhaul of the Pocket Network protocol](future/README.md), including a departure from Tendermint. v1 will allow Pocket to build its own network from the ground up, optimized for its specific use case, rather than being restricted by any limitations that come with building on Tendermint. v1 will bring major changes to four different "modules": Utility, Consensus, Peer-to-Peer, and Persistence.
+
 ## Validator
 
-**Validators** are responsible for committing new blocks in the blockchain. These **validators** participate in the consensus protocol by broadcasting votes which contain cryptographic signatures signed by each **validator’s** private key.
+Special nodes that are responsible for committing new blocks in the blockchain. Validators participate in the consensus protocol by broadcasting votes which contain cryptographic signatures signed by each validator's private key.
+
+Validators stake POKT into the protocol to participate in Servicing and Consensus, just like Servicers, but because they are able to be block proposers, they can earn a slightly higher reward amount, as determined by the [ProposerAllocation](protocol-parameters.md#proposerallocation) parameter.
+
+A Servicer that is in the top 1,000 node stakes on the network may be a Validator.
+
+## WAGMI
+
+Weighted Annual Gross Max Inflation (WAGMI) is an inflation control mechanism for the Pocket Network that was established via [PUP-11](https://forum.pokt.network/t/pup-11-wagmi-inflation/), and further defined via [PUP-13](https://forum.pokt.network/t/pup-13-initial-wagmi-parameters/). Together, these proposals established a target annual inflation rate, or WAGMI rate, in POKT, from which the [RelaysToTokensMultiplier](protocol-parameters.md#relaystotokensmultiplier) parameter (which determines the amount of POKT earned per relay serviced) is adjusted dynamically. The proposals also laid out a step-down in target inflation rate from 100% to 50% over a period of time.
+
+## Whitelisted Chains
+
+In the Pocket Portal, a list of the blockchains that an application can connect to. For example, if four chains are whitelisted, there will be four endpoints generated in that application.
+
+## Whitelisted Contracts
+
+In the Pocket Portal, a list of smart contract addresses that can access an application endpoint. Note that this is only available for EVM-compatible chains.
```

Comment:
~~a list~~ an exclusive list

Reply:
Why the focus on "exclusive" in all of these places?